### ISSUE TITLE
real bout fatal fury sound issue?

### ISSUE THREAD

**Issue by xenyou** [MiSTer-devel/NeoGeo_MiSTer#211]
I don't exactly know if this is a issue but I'll report it.

it seems that the volume balance of Mai's BGM is weird.

### mister fpga (neogeo core)
some sound is little loud.

https://youtu.be/6KXYI3J-U8U?si=tVByWrn0I4Xd9vJc&t=58

### mvs (MV-1B. real hardware of mine)
not loud.

https://youtu.be/cKeXD93-wyc?si=7Wq3Z7kt4hV-oHwS&t=84

### mvs (famous game center of japan)
not loud.

https://www.youtube.com/watch?v=X245vhXOwTs&t=3124s

### ACA Neogeo
this one is similar with mister's one. the same sound is little loud.

https://youtu.be/pJbmZhXB5JI?si=EXx7NCGANj3Wlc2s&t=58

Does it depend on the model of MVS?

**Comment by Toryalai1** [comment]
Yeah I can confirm this issue. There is a sound channel that is too loud.

**Comment by Toryalai1** [comment]
We finally have some data thanks to Artemio and team. I'm just going to paste in information from him and maybe developers can jump in.

**AES vs AES:
You can see that accoring to measurement ADPCM-A is too low (or the rest are too loud if you are relative to that) and that curves are filtered differently
with ADPCM-B too loud in the high end**

![image](https://github.com/MiSTer-devel/NeoGeo_MiSTer/assets/82502378/1b307082-c749-4840-810c-b8717d33921c)

**MVS 1FS vs Mister:
FM is high by about 3db, and ADPCM B is high by about 1.7 db**

![image](https://github.com/MiSTer-devel/NeoGeo_MiSTer/assets/82502378/4176f829-51cd-45a8-9277-b9338f8e04b6)

![image](https://github.com/MiSTer-devel/NeoGeo_MiSTer/assets/82502378/a4ce2018-b4b0-4dfd-bc2c-6db6e3c43d0d)

![image](https://github.com/MiSTer-devel/NeoGeo_MiSTer/assets/82502378/3d3b55e6-1a86-4509-b2a1-f7b65493729a)


Here are the recordings: 
https://drive.google.com/drive/folders/1gZP5rWWdjabl7LnLB3SplF2QpiX_uHxa

@paulb-nl, I know that you did some fixing to the audio before, do you think this would be enough information? :)

**Comment by MP2E** [comment]
I adjusted the volume of ADPCM-A to match closely to an AES console and it's sounding better in my opinion.

![DA__ALL_AES-headphone-FULL-192khz_vs_mister_neogeo_0000](https://github.com/user-attachments/assets/6febdc5a-29ee-42a3-b901-4808d8285ea1)

Here's the build if anyone wants to check it out. Will post the changes as a pull request soon.

[NeoGeo_20240824.rbf.zip](https://github.com/user-attachments/files/16736234/NeoGeo_20240824.rbf.zip)

Unfortunately, as you can see I haven't changed the filtering at all at this point.

**Comment by okami2012** [comment]
> Yeah I can confirm this issue. There is a sound channel that is too loud.

Oh i 'm sorry! This issue is similar to the one I reported a few days ago. I thought it was caused by game speed issues, and my game background music sound didn't sound as powerful as the real MVS sound, so I mistakenly thought there was a problem with the game speed. I downloaded the latest unstable version, which is slightly different from the real board, but it has improved a bit!
